package.json

```diff
@@ -69,7 +63,6 @@
     "@netlify/eslint-config-node": "^4.1.2",
     "ava": "^3.0.0",
     "from2-string": "^1.1.0",
-    "husky": "^4.3.8",
     "nock": "^13.0.0",
     "npm-run-all": "^4.1.5",
```

Comment:
@ehmicky @erezrokah Might be worth droping npm-run-all from the direct devDependencies in your projects since it's included in @netlify/eslint-config-node. Just thinking out loud :)

Reply:
I believe you are correct.

Binary symlinks seem to be installed by `npm` to `node_modules/.bin/` for both direct and indirect dependencies. That being said, I am 100% sure this is the case on other package managers, but we could try it out and see if anyone mentions it not working.